File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.js
function solution(price) {
    if (price >= 500000) {
        return parseInt(price - (price * 0.2));
    } else if (price >= 300000) {
        return parseInt(price - (price * 0.1));
    } else if (price >= 100000) {
        return parseInt(price - (price * 0.05));
    } else {
        return price;
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    answer = 0
    
    if price >= 500000:
        answer = int(price - (price * 0.2))
    elif price >= 300000:
        answer = int(price - (price * 0.1))
    elif price >= 100000:
        answer = int(price - (price * 0.05))
    else:
        answer = price
        
    return answer